docs/guide/blog/index.md

```diff
@@ -22,6 +22,14 @@ By completing this tutorial, you will learn about:
 
 **Note:** All the functions in this chapter can be scaffolded with a single command but instead we will learn how to add each functionality individually. 
 
+## Prerequisites 
+
+This series of tutorials is based on a specific version of Starport, so to install Starport v0.19.1 use the following command:
```

Comment:
```suggestion
This tutorial is based on a specific version of Starport, so to install Starport v0.19.1 use the following command:
```

Reply:
I think @barriebyron meant - the 3 tutorials on Blog we have.